### PR TITLE
Added documentation for manual pod deletion

### PR DESCRIPTION
**Pull request by ppatierno**
### Type of change

- Documentation

### Description

This PR adds documentation related to manual pod (and pvc) deletion brought by #949. 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/proc-manual-delete-pod-pvc-kafka.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+On {KubernetesName} this can be done using `kubectl annotate`:
+[source,shell,subs=+quotes]
+kubectl annotate pod _cluster-name_-kafka-_index_ cluster.operator.strimzi.io/delete-pod-and-pvc=true
+endif::Kubernetes[]
++
+On {OpenShiftName} this can be done using `oc annotate`:
+[source,shell,subs=+quotes]
+oc annotate pod _cluster-name_-kafka-_index_ cluster.operator.strimzi.io/delete-pod-and-pvc=true
++
+. Wait for next reconciliation and the annotated pod will be deleted and then recreated.
```
Comment (scholzj) [inline documentation/book/proc-manual-delete-pod-pvc-kafka.adoc]: _and the annotated pod with the underlying PVC will be deleted_

File: documentation/book/proc-manual-delete-pod-pvc-kafka.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
```
Comment (laidan6000) [inline documentation/book/proc-manual-delete-pod-pvc-kafka.adoc]: _Please use the following content, including reworded title:_

## Delete Kafka nodes manually

This procedure describes how to delete an existing Kafka node by using an {ProductPlatformName} annotation. Deleting a Kafka node consists of deleting both the Pod on which the Kafka broker is running and the related `PersistentVolumeClaim` (if the cluster was deployed with persistent storage). After deletion, the Pod and its related `PersistentVolumeClaim` are recreated automatically.

Warning: Deleting a `PersistentVolumeClaim` can cause permanent data loss. The following procedure should only be performed if you have encountered storage issues.

### Prerequisites

* A running Kafka cluster.
* A running Cluster Operator.

### Procedure

1. Find the name of the Pod that you want to delete.
 
For example, if the cluster is named cluster-name, the pods are named cluster-name-kafka-index, where index starts at zero and ends at the total number of replicas.

2. Annotate the Pod resource in {ProductPlatformName}.

  On {OpenShiftName}, use `oc annotate`:

  ``oc annotate pod cluster-name-kafka-index cluster.operator.strimzi.io/delete-pod-and-pvc=true``

3. Wait for the next reconciliation, when the annotated pod with the underlying persistent volume claim will be deleted and then recreated.

### Additional resources
For more information on deploying the Cluster Operator, see [cluster-operator-str].

For more information on deploying the Kafka cluster, see ] and xref:deploying-kafka-cluster-kubernetes-str[.

File: documentation/book/proc-manual-delete-pod-pvc-zookeeper.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
```
Comment (laidan6000) [inline documentation/book/proc-manual-delete-pod-pvc-zookeeper.adoc]: _Please use the following content, including reworded title:_

##  Delete Zookeeper nodes manually
This procedure describes how to delete an existing Zookeeper node by using an {ProductPlatformName} annotation. Deleting a Zookeeper node consists of deleting both the Pod on which Zookeeper is running and the related `PersistentVolumeClaim` (if the cluster was deployed with persistent storage). After deletion, the Pod and its related `PersistentVolumeClaim` are recreated automatically.

Warning: Deleting a `PersistentVolumeClaim` can cause permanent data loss. The following procedure should only be performed if you have encountered storage issues.

### Prerequisites

* A running Zookeeper cluster.
* A running Cluster Operator.

### Procedure

1. Find the name of the Pod that you want to delete.

  For example, if the cluster is named cluster-name, the pods are named cluster-name-zookeeper-index, where index starts at zero and ends at the total number of replicas.

2. Annotate the Pod resource in {ProductPlatformName}.

+ On {OpenShiftName} use `oc annotate`:

  ``oc annotate pod cluster-name-zookeeper-index cluster.operator.strimzi.io/delete-pod-and-pvc=true``

3. Wait for the next reconciliation, when the annotated pod with the underlying persistent volume claim will be deleted and then recreated.

### Additional resources
For more information on deploying the Cluster Operator, see [cluster-operator-str].

For more information on deploying the Zookeeper cluster, see ] and xref:deploying-zookeeper-cluster-kubernetes-str[.

**Comment by laidan6000** [comment]
I checked the edits. Everything looks good to me.

**Pull request merged**